romana-install/roles/cache/apply/tasks/repo_ubuntu_trusty.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: Synchronize apt cache to hosts
+  synchronize: src="{{ cache_dir }}/{{ ansible_distribution|lower }}_{{ ansible_distribution_release|lower }}_apt/" dest="/var/cache/apt/archives/"
```

Comment:
It could. `group_vars/all/internal` seems like a sensible thing to create and put variables used exclusively within the playbooks to keep it cleaner.